Makefile

```diff
@@ -13,10 +13,10 @@ stop-postgres:
 clean:
 	find . -type f -regex ".*\.py[co]$$" -delete
 	find . -type d -name "__pycache__" -delete
-	rm -rf dispatcher.egg-info/
+	rm -rf ansible_dispatcher.egg-info/
```

Comment:
Instead of tracking things manually, it's usually easier to just invoke `git clean`.